omigami/plotting.py

```diff
@@ -1,60 +1,153 @@
+                                     sort_by_score: bool = True, draw_indices: bool = False,
+                                     molecule_image_size: List[int] = [200, 200],
+                                     substructure_highlight: str = "") -> PngImageFile:
+        """
+        Generate a grid image representation of the hits returned from Spec2Vec and MS2DeepScore outputs.
+        All structures passed MUST have valid smiles or inchi representations.
+
+        Parameters:
+        ----------
+        spectra_matches: DataFrame
+            DataFrame resulting from either Spec2Vec or MS2DeepScore. Need to feature smiles or inchi, score and compound_name as columns.
+        representation: str = 'smiles' or 'inchi'
+            The representation of the molecules found in the provided dataframe
+        sort_by_score: bool = True
+            If true sorts the dataframe by the score column
+        draw_indices: bool = False
+            If true draws the indices of the atoms
+        molecule_image_size: List[int, int] = [200, 200]
+            The size of every individual image of a molecule. Need to be provided as a list with two ints
+        substructure_highlight: str = None
+            Needs to be a molecule substructure represented as a smiles.
+
+        Returns:
+        -------
+            A Plot showing the structure of the passed smiles/inchis
+        """
+
+        self._validate_data(spectra_matches, representation)
+
+        if sort_by_score:
+            spectra_matches = spectra_matches.sort_values('score', ascending=True)
+
+        spectra_matches = self._clean_matches(spectra_matches, representation)
+
+        substructure = Chem.MolFromSmarts(substructure_highlight)
+        mol_render_list = []
+        highlight_bonds = []
+
+        for structure in spectra_matches[representation]:
+
+            if representation == 'smiles':
+                molecule = Chem.MolFromSmiles(structure)
+            elif representation == 'inchi':
+                molecule = Chem.MolFromInchi(structure)
+
+            substructure_matches = self._get_bonds_to_highlight(molecule, substructure)
+            highlight_bonds.append(substructure_matches)
+
+            if draw_indices:
+                molecule = self._mol_with_atom_index(molecule)
+
+            mol_render_list.append(molecule)
+
+        image = Draw.MolsToGridImage(mol_render_list, subImgSize=molecule_image_size,
+                                     legends=spectra_matches.compound_name.tolist(),
+                                     highlightBondLists=highlight_bonds)
+        return image
+
+    @staticmethod
+    def plot_classyfire_result(smiles_list: List[str], color="g"):
+        """TODO: Ask joe what those classifiers actually classfi"""
+        class_stats = dict()
+        for smiles in smiles_list:
+            try:
+                classyfire_result = requests.get('https://gnps-structure.ucsd.edu/classyfire?smiles=' + smiles).json()
```

Comment:
maybe we can add these urls in a config file?? instead of having them hardcoded here. what do you think?